src/converse-register.js

```diff
@@ -241,6 +245,25 @@
                     return false;
                 },
 
+                chooseProvider: function (ev) {
```

Comment:
@jcbrand I will need another method, because in case registration_domain is mentioned by user, I'll have to invoke that function. onProviderChoosen() has been kept specifically for binding it to the `submit` event. How specifically do you want the onProviderChoosen() to be called? Firstly it requires an event as a parameter. Secondly, if I do call it separately, should I wrap codes specific to the `registration_domain` config in an if, and do the necessary? Waiting for your inputs on this...